examples/non_cartesian_reconstruction.py

```diff
@@ -49,7 +49,7 @@
 # We then reconstruct the zero order solution as a baseline
 
 # Get the locations of the kspace samples and the associated observations
-fourier_op = NFFT(samples=kspace_loc, shape=image.shape)
+fourier_op = NonCartesianFFT(samples=kspace_loc, shape=image.shape)
```

Comment:
add the implementation parameter even if it is the default value

Reply:
Sure